scripts/qa_crowbarsetup.sh

```diff
@@ -1350,6 +1350,22 @@ function custom_configuration()
             if [ -n "$networkingplugin" ] ; then
                 proposal_set_value neutron default "['attributes']['neutron']['networking_plugin']" "'$networkingplugin'"
             fi
+            # For Cloud5+, proposal attribute names changed
+            if iscloudver 5plus; then
+                if [ "$networkingplugin" = openvswitch ] ; then
+                    proposal_set_value neutron default "['attributes']['neutron']['networking_plugin']" "'ml2'"
+                    proposal_set_value neutron default "['attributes']['neutron']['ml2_type_drivers']" "['gre','vlan']"
```

Comment:
> Shouldn't all this be something we can change with parameters?

Yes. But as it is now, we don't have to change something on the Jenkins side and can get the ml2 things merged as fast as possible. I would like to have new parameters in an extra commit. And then we should test openvswitch with vlan and gre separately. Does this make sense?